### PR TITLE
Merge release candidate dev into master before submitting to CRAN

### PR DESCRIPTION
**Pull request by jooolia**
Hello,

As described in Issue #102 we are hoping to submit a new version of rsnps to CRAN that fixes many issues we have encountered recently. 

@sckott would you mind having a look at this PR to see if things look ok for this go-around? 

Thanks! Julia and Sina

### CODE REVIEW

**Comment by sckott** [comment]
Sure thing

File: DESCRIPTION [hunk 1]
```diff
@@ -30,11 +32,14 @@ Imports:
...
-    jsonlite
+    jsonlite,
+    RJSONIO,
+    httr
 Suggests:
     testthat,
-    knitr
-RoxygenNote: 6.1.1
+    knitr,
+    markdown
```
Comment (sckott) [inline DESCRIPTION:41]: did you mean to use the package `rmarkdown` here?

**Comment by jooolia** [comment]
Thanks a lot @sckott. We will work on addressing all of your helpful comments.

File: R/ncbi_snp_api.R [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+#' - MAF: The minor allele frequency of the SNP, given it is an SNV.
+#' This is drawn from the current global reference population used by NCBI (GnomAD).
+#' - AncestralAllele: allele as described in the current assembly
+#' - VariationAllele: difference to the current assembly
+#' - seqname - Chromosome RefSeq reference.
+#' - hgvs -  full hgvs notation for variant
+#' - assembly - which assembly was used for the annotations
+#' - ref_seq - sequence in reference assembly
+#'
+#' @seealso [ncbi_snp_query()]
```
[the remark this answers is not in the document]
Comment (jooolia) [reply R/ncbi_snp_api.R:177]: Yes agreed, will remove.

File: R/ncbi_snp_api.R [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+#' ncbi_snp_query("rs121909001")
+#' ncbi_snp_query("rs1837253")
+#' ncbi_snp_query("rs1209415715")
+#' ncbi_snp_query("rs111068718")
+#' ncbi_snp_query(SNPs='rs9970807')
+#'
+#' ncbi_snp_query("rs121909001")
+#' ncbi_snp_query("rs121909001", verbose = TRUE)
+#' }
+ncbi_snp_query <- function(SNPs, ...) {
```
[the remark this answers is not in the document]
Comment (jooolia) [reply R/ncbi_snp_api.R:202]: True. This update seems not to be in dev, but we will update this to match what we say in NEWS as that is what we wanted.

File: R/ncbi_snp_api.R [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+  
+  out <- as.data.frame(matrix(0, nrow = length(SNPs_num), ncol = 15))
+  names(out) <- c("Query", "Chromosome", "BP", "Class", "rsid", "Gene", "Alleles", "AncestralAllele", "VariationAllele", "seqname", "hgvs", "assembly", "ref_seq", "Minor", "MAF")
+  
+  ## as far as I understand from https://api.ncbi.nlm.nih.gov/variation/v0/#/RefSNP/ we
+  ## can only send one query at a time and max 1 per second.
+  for (i in seq_along(SNPs_num)) {
+    
+    variant.url <- paste0("https://api.ncbi.nlm.nih.gov/variation/v0/refsnp/", SNPs_num[i])
+    variant.response <-  httr::GET(variant.url)
```
[the remark this answers is not in the document]
Comment (jooolia) [reply R/ncbi_snp_api.R:227]: Interesting, I admit I know very little about what parameter would be passed here.  Do you have any examples of what people may pass here? Just curious about this.

File: DESCRIPTION (hunk 1 above)
Comment (jooolia) [reply DESCRIPTION:41]: This was to resolve an issue we had when running `devtools::check_win_devel()` (see comment here: https://github.com/ropensci/rsnps/issues/102#issuecomment-658233790).

File: R/ncbi_snp_api.R (hunk 4 above)
[the remark this answers is not in the document]
Comment (sckott) [reply R/ncbi_snp_api.R:227]: e.g. `httr::verbose()` `httr::timeout()`, see `?httr::config` - You could just remove the `...` from the function definition though and then users if they wanted to set curl options could still do so by doing `httr::set_config`/`httr::with_config`

**Comment by jooolia** [comment]
Ok thanks for the explanation! That helps!

**Comment by jooolia** [comment]
Hi @sinarueeger, 
We are getting 7 failed tests because of "SSL certificate problem: certificate has expired" type errors. This is the reason for the failed Travis CI build and I can also reproduce this locally. I will re-run the tests tomorrow to see if it still remains an issue.

**Comment by sinarueeger** [comment]
Hi @jooolia , the ssl error is because of https://opensnp.org/ I think. I asked them to extend their ssl certificate: https://gitter.im/openSNP/.

**Comment by jooolia** [comment]
ok well this morning it seems to be ok. :)

**Comment by jooolia** [comment]
Hi @sinarueeger,
I think that we have addressed all the current issues (and I cannot get the travis-ci/push test to be redone but I think it should now be ok again for the moment, but the most recent win-builder https://win-builder.r-project.org/0D7lBEzduf98/ is as expected). Are you ok with merging this branch into master and then we submit (as in #102). Anything else that you would like to see done?

Thanks, Julia

**Comment by sinarueeger** [comment]
Hi @joolia, thanks! This looks ok to me :-)

**Pull request merged**